components/Header/index.js

```diff
@@ -5,32 +5,34 @@ const metricsData = require("../../data/metrics.json");
 import subDays from 'date-fns/subDays';
 import format from 'date-fns/format'
 
-const getQueryParams = () => {
-  let urlString = document.URL;
-  if(urlString && urlString.includes('?')){
-    //console.log(urlString.split('?'))
-    let paramString = urlString.split('?')[1]; 
-    let queryString = paramString.split('&'); 
-    const params = {
-      id: (queryString[0]).replace("id=", ""),
-      //date: (queryString[1]).replace("date=", ""),
-      selectedRange: queryString[1].replace("range=", "")
-    };
-    return params;
+const getQueryParams = (query) => {
+  const { id, range } = query;
+  
+  switch (range) {
+    case 'today':
+      retun 1;
+    case 'yesterday':
+      return 2;
+    case 'week':
+      return 3;
+    case 'month':
+      return 4;
+    default:
+      return 1;
   }
-  const obj = metricsData.find(o => o.date === String(date));
```

Comment:
this will get the metrics data for selected range

Reply:
dont we have to consider the id?

Reply:
What id?